libfec/src/fec.c

```diff
@@ -0,0 +1,65 @@
+/* Utility routines for FEC support
+ * Copyright 2004, Phil Karn, KA9Q
+ */
+
+#include "fec.h"
+
+unsigned char Partab[256];
+int P_init;
+
+/* Create 256-entry odd-parity lookup table
+ * Needed only on non-ia32 machines
+ */
+void partab_init(void){
+  int i,cnt,ti;
+
+  /* Initialize parity lookup table */
+  for(i=0;i<256;i++){
+    cnt = 0;
+    ti = i;
+    while(ti){
+      if(ti & 1)
+	cnt++;
+      ti >>= 1;
+    }
+    Partab[i] = cnt & 1;
+  }
+  P_init=1;
+}
+
+/* Lookup table giving count of 1 bits for integers 0-255 */
+int Bitcnt[] = {
```

Comment:
Delete this.  There are ways to do a fast population count without a huge table, but it looks like it's not even used anyway.